lib/plugins/aws/configCredentials.js

```diff
@@ -51,17 +52,18 @@ class AwsConfigCredentials {
       );
     }
 
-    this.serverless.cli.log('Setting up AWS...');
+    legacy.log('Setting up AWS...');
 
     const profiles = await credentials.resolveFileProfiles();
     if (profiles.has(this.options.profile)) {
       // Only update the profile if the overwrite flag was set
       if (!this.options.overwrite) {
-        const message = [
-          `Failed! ~/.aws/credentials already has a "${this.options.profile}" profile.`,
-          ' Use the overwrite flag ("-o" or "--overwrite") to force the update',
-        ].join('');
-        this.serverless.cli.log(message);
+        legacy.log(
+          `Failed! ~/.aws/credentials already has a "${this.options.profile}" profile. Use the overwrite flag ("-o" or "--overwrite") to force the update`
+        );
+        log.notice.skip(
+          `Profile "${this.options.profile}" is already configured in ~/.aws/credentials. Use the overwrite flag ("-o" or "--overwrite") to force the update.`
```

Comment:
This feels more as an error. I would I think convert it to `ServerlessError` exception